integration/tools/validation/src/main/java/alluxio/cli/fuse/ValidateRead.java

```diff
@@ -0,0 +1,246 @@
+package alluxio.cli.fuse;
+
+import static alluxio.cli.fuse.CorrectnessValidationUtils.BUFFER_SIZES;
+import static alluxio.cli.fuse.CorrectnessValidationUtils.DATA_INCONSISTENCY_FORMAT;
+import static alluxio.cli.fuse.CorrectnessValidationUtils.FILE_SIZES;
+import static alluxio.cli.fuse.CorrectnessValidationUtils.RANDOM;
```

Comment:
May be better to put it in a file with name `Constants` for shared constants
Utils usually put the shared values for utils methods but not for other class methods